src/Statistics/include/Statistics/Histogram.h

```diff
@@ -36,6 +36,12 @@ struct Histogram {
 // estimates the risk score using `HistogramRiskScore` and returns the histogram
 // which minimizes it. The histogram will not own the data.
 [[nodiscard]] std::optional<Histogram> BuildHistogram(absl::Span<const uint64_t> data);
+
+// The functions aids histogram rendering.
+// Takes two positive intergers, returns a vector `result` of non-negative integers s.t.
+// their sum equals to `histogram_width` and for all `i` and `j` `max(result[i] - result[j]) <= 1`
+// `histogram_width` represents the width of the histogarm in pixels.
+[[nodiscard]] std::vector<int> GetBinWidth(size_t number_of_bins, int histogram_width);
```

Comment:
I would rename "get" to something else, such as "compute". "Get" suggests a getter, i.e., something that simply returns an existing value, while actual work is done by this function.